mlir/utils/performance/conv-configs

```diff
@@ -1,43 +1,43 @@
 # This section of the file comes from resnet50-miopen-configs
--n 256 -c 1024 -H 14 -W 14 -k 2048 -y 1 -x 1 -p 0 -q 0 -u 2 -v 2 -l 1 -j 1 -m conv -g 1 -t 1
```

Comment:
Swapped "-m conv" and "-g 1" when I was trying to make all the input and output configs match, since perfRunner.py did a simple string-compare.  Eventually made toCommandLine and canonicalised them all at runtime.